packages/terra-functional-testing/src/config/utils/getIpAddress.js

```diff
@@ -0,0 +1,12 @@
+const ip = require('ip');
+const os = require('os');
+
+module.exports = () => {
+  const utun = Object.entries(os.networkInterfaces()).find(([key, networkInterface]) => key.includes('utun') && networkInterface[0] && networkInterface[0].family === 'IPv4');
+
+  if (utun && utun[1]) {
+    return utun[1][0].address;
+  }
+
+  return ip.address();
+};
```

Comment:
Nice, I like this being split out into a util file.

Reply:
When in Rome. I just copied the pattern established by getCapabilities.
![](https://media.giphy.com/media/2stUfIoQ6jmi4/giphy.gif)